### PR TITLE
Add TOC to landing page. Insert link to Google groups

### PR DESCRIPTION
**Pull request by santoshshanbhogue**
Change the landing page to make it more informative. Users now need not click on each folder to see what lurks beneath. Add a link to google groups so that users can post questions for help

Use the word "cloud" because that makes it clear to most users that there is web-version that can be used in minutes

### CODE REVIEW

**Comment by speth** [comment]
The extended directions and table look mostly fine. Instead of "you can now run Cantera in the cloud", I would say "you can now *try* ...", to emphasize that this is not a full fledged solution.

Linking to the Google Group is fine, but I think I would point users to the documentation first (and specifically, the documentation for the Python module). We already get enough questions on the Group from users who apparently haven't read any of it.

A couple of formatting suggestions: The new headings should be second level (`##`), not third (`###`), and I think the individual examples should be bulleted instead of numbered.

**Comment by santoshshanbhogue** [comment]
The extended directions and table look mostly fine. Instead of "you can now run Cantera in the cloud", I would say "you can now try ...", to emphasize that this is not a full fledged solution.
-- Done. Now emphasized that new users could download Cantera as well

Linking to the Google Group is fine, but I think I would point users to the documentation first (and specifically, the documentation for the Python module). We already get enough questions on the Group from users who apparently haven't read any of it.
-- Done

A couple of formatting suggestions: The new headings should be second level (##), not third (###), and I think the individual examples should be bulleted instead of numbered.
-- Done

**Comment by santoshshanbhogue** [comment]
Second level indentations automatically produce a horizontal rule

**Comment by santoshshanbhogue** [comment]
Just so that no-one goes all-out-nuts with myBinder

File: README.md [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+    * [Equilibrium flame temperature calculator](https://github.com/Cantera/cantera-jupyter/blob/master/thermo/flame_temperature.ipynb)
+
+* Flame Simulations
+
+    * [Flame speed calculator with sensitivity analysis](https://github.com/Cantera/cantera-jupyter/blob/master/flames/flame_speed_with_sensitivity_analysis.ipynb)
+    * [Counter-flow twin premixed flame simulator](https://github.com/Cantera/cantera-jupyter/blob/master/flames/twin_premixed_flame_axisymmetric.ipynb)
+    
+## Frequently Asked Questions
+
+**How do I use Cantera with Python**
```
Comment (speth) [inline README.md]: missing question mark

File: README.md [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+
+To learn how to use Cantera with Python, click [here](http://cantera.github.io/docs/sphinx/html/cython/index.html). For more advanced uses of Cantera, the complete documentation can be found [here](http://cantera.github.io/docs/sphinx/html/index.html)
+
+**Can I forgo installing Cantera locally and just use Cantera in the cloud every time**
```
Comment (speth) [inline README.md]: missing question mark

File: README.md [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+* Flame Simulations
+
+    * [Flame speed calculator with sensitivity analysis](https://github.com/Cantera/cantera-jupyter/blob/master/flames/flame_speed_with_sensitivity_analysis.ipynb)
+    * [Counter-flow twin premixed flame simulator](https://github.com/Cantera/cantera-jupyter/blob/master/flames/twin_premixed_flame_axisymmetric.ipynb)
+    
+## Frequently Asked Questions
+
+**How do I use Cantera with Python**
+
+To learn how to use Cantera with Python, click [here](http://cantera.github.io/docs/sphinx/html/cython/index.html). For more advanced uses of Cantera, the complete documentation can be found [here](http://cantera.github.io/docs/sphinx/html/index.html)
```
Comment (speth) [inline README.md]: need period after `[here]`.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+
+**Can I forgo installing Cantera locally and just use Cantera in the cloud every time**
+
+The problem with using Cantera with Binder is that there is no way for you to save your work. You can upload/download files in a session, but once the session is over (you close your browser window), you lose all your work. You thus cannot save your modified iPython notebooks. 
+
+**I still can't figure something out. Who do I ask?**
+
+If you have more questions, need help with something, or have any suggestions, please visit [Cantera Google Groups Page](https://groups.google.com/forum/#!forum/cantera-users) and create a post
```
Comment (speth) [inline README.md]: period at the end.

**Comment by santoshshanbhogue** [comment]
Added two question marks, two periods and one article.

**Pull request closed without merging**